internal/driver/driver_test.go

```diff
@@ -33,6 +38,147 @@ func TestDriver(t *testing.T) {
 	require.NoError(t, err)
 	t.Cleanup(func() { require.NoError(t, c.Close()) })
 
-	// TODO https://github.com/FerretDB/FerretDB/issues/4146
-	_ = c
+	dbName := "TestDriver"
+
+	var lsid bson.Binary
+
+	lsid = bson.Binary{
+		B: []byte{
+			0xa3, 0x19, 0xf2, 0xb4, 0xa1, 0x75, 0x40, 0xc7,
+			0xb8, 0xe7, 0xa3, 0xa3, 0x2e, 0xc2, 0x56, 0xbe,
+		},
+		Subtype: bsonproto.BinaryUUID,
+	}
+
+	t.Run("Drop", func(t *testing.T) {
+		doc := must.NotFail(bson.NewDocument(
+			"dropDatabase", int32(1),
+			"lsid", must.NotFail(bson.NewDocument("id", lsid)),
+			"$db", dbName,
+		))
+
+		body, err := wire.NewOpMsg(must.NotFail(doc.Encode()))
+		require.NoError(t, err)
+
+		_, _, err = c.Request(ctx, new(wire.MsgHeader), body)
```

Comment:
Let's check the response

Reply:
I didn't check it here, as it's mostly here for cleanup, the results may vary

Reply:
At the very least, we should check `{"ok": 1}`